src/mixer/samplerbank.cpp

```diff
@@ -105,37 +123,42 @@ void SamplerBank::slotLoadSamplerBank(double v) {
         return;
     }
 
+    if (!loadSamplerBankFromPath(samplerBankPath)) {
+        QMessageBox::warning(NULL,
+                      tr("Error Reading Sampler Bank"),
+                      tr("Could not open the sampler bank file '%1'.")
+                      .arg(samplerBankPath));
+    }
+}
+
+bool SamplerBank::loadSamplerBankFromPath(const QString& samplerBankPath) {
     // The user has picked a new directory via a file dialog. This means the
     // system sandboxer (if we are sandboxed) has granted us permission to this
     // folder. We don't need access to this file on a regular basis so we do not
     // register a security bookmark.
 
+    if (m_pPlayerManager == nullptr) {
+        qWarning() << "SamplerBank::loadSamplerBankFromPath called with no PlayerManager";
```

Comment:
same as above, that is probably something internal, code-related, so should be debug-asserted?